pkg/config/cluster.go

```diff
@@ -0,0 +1,86 @@
+package config
+
+import (
+	"reflect"
+
+	"github.com/ethersphere/beekeeper/pkg/bee"
+)
+
+// Cluster represents cluster configuration
+type Cluster struct {
+	// parrent to inherit settings from
+	*Inherit `yaml:",inline"`
+	// Cluster configuration
+	Name                *string           `yaml:"name"`
```

Comment:
just wondering - why the use of pointers here?

Reply:
Methods use reflection. Implement with pointers was straightforward.